package.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Use 4 spaces for tabs instead of 2. Did you `npm run pretty`?

Reply:
Yes, maybe it is missconfigured

Reply:
Found it, wrong path in package.json prettier command